quinn-udp/src/windows.rs

```diff
@@ -107,16 +113,33 @@ impl UdpSocketState {
             )?;
         }
 
-        // Opportunistically try to enable GRO
-        _ = set_socket_option(
-            &*socket.0,
-            WinSock::IPPROTO_UDP,
-            WinSock::UDP_RECV_MAX_COALESCED_SIZE,
-            // u32 per
-            // https://learn.microsoft.com/en-us/windows/win32/winsock/ipproto-udp-socket-options.
-            // Choice of 2^16 - 1 inspired by msquic.
-            u16::MAX as u32,
-        );
+        match &*IS_WINDOWS_ON_ARM {
+            Ok(true) => {
+                // Bug on Windows on ARM, not receiving `UDP_COALESCED_INFO` `CMSG`
+                // when _Virtual Machine Platform_ feature enabled. See
+                // <https://github.com/quinn-rs/quinn/issues/2041> for details.
+                debug!("detected Windows on ARM host thus not enabling URO")
+            }
+            Ok(false) => {
+                // Opportunistically try to enable URO
+                let result = set_socket_option(
+                    &*socket.0,
+                    WinSock::IPPROTO_UDP,
+                    WinSock::UDP_RECV_MAX_COALESCED_SIZE,
+                    // u32 per
+                    // https://learn.microsoft.com/en-us/windows/win32/winsock/ipproto-udp-socket-options.
+                    // Choice of 2^16 - 1 inspired by msquic.
+                    u16::MAX as u32,
+                );
+
+                if let Err(e) = result {
+                    debug!("failed to enable URO: {}", e);
+                }
+            }
+            Err(e) => {
+                debug!("failed to detect host system thus not enabling URO: {}", e);
```

Comment:
Nit: inline format argument (also above).

Reply:
Unfortunately that would make `e` unused if neither feature `tracing` nor `direct-log` is enabled:

https://github.com/quinn-rs/quinn/blob/f18890960d7911739b5ed9402e85e8f8ad02b834/quinn-udp/src/lib.rs#L56-L77

See for example CI failure:

https://github.com/quinn-rs/quinn/actions/runs/12068811973/job/33654728590?pr=2071